contributors/design-proposals/storage/svcacct-token-volume-source.md

```diff
@@ -80,6 +80,9 @@ type ServiceAccountTokenProjection struct {
   ExpirationSeconds int64
   // Path is the relative path of the file to project the token into.
   Path string
+  // IsContainerLevel determines if the token should include information about
+  // which container the request comes from.
+  IsContainerLevel bool
```

Comment:
this would mean the kubelet would have to fan out and maintain N tokens for a pod with N containers... is that what we want?

Reply:
@liggitt  For most use cases who don't need container-level serviceaccount feature, they just launch a token projection volume with IsContainerLevel being false.  In this case, all the containers could share single serviceaccount token.